src/main/java/io/numaproj/numaflow/common/GrpcServerConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.numaproj.numaflow.common;
+
+import io.numaproj.numaflow.function.Function;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class GrpcServerConfig {
+    private String socketPath;
+    private int maxMessageSize;
+
+    public GrpcServerConfig(String socketPath, int maxMessageSize) {
```

Comment:
very nit: we could also remove the constructor using Lombok @Data annotation.

Reply:
I checked. @Data doesn't provide all args constructor.. it provides @RequiredArgsConstructor